packages/story-editor/src/components/floatingMenu/elements/shared/toggleButton.js

```diff
@@ -34,6 +34,7 @@ const ToggleButton = forwardRef(function ToggleButton(
       ? {
           type: isToggled ? BUTTON_TYPES.SECONDARY : BUTTON_TYPES.TERTIARY,
           'aria-pressed': isToggled,
+          role: 'button',
```

Comment:
in a `ContextMenu` the children default to `menuitem` and `aria-pressed` isn't a valid attribute for that, so let's make sure they are buttons.